docs/upgrade/_common/upgrade-guide-from-5.0-to-2022.1-ubuntu-and-debian.rst

```diff
@@ -0,0 +1,204 @@
+
+**During** the rolling upgrade, it is highly recommended:
+
+* Not to use new 2022.1 features.
+* Not to run administration functions, like repairs, refresh, rebuild or add or remove nodes. See :doc:`here </operating-scylla/manager/2.1/sctool>` for suspending Scylla Manager's scheduled or running repairs.
+* Not to apply schema changes.
+
+
+.. include:: /upgrade/_common/upgrade_to_2022_warning.rst
+  
+Upgrade Steps
+=============
+
+Check the cluster schema
+--------------------------
+Make sure that all nodes have the schema synched before the upgrade. The upgrade will fail if there is a schema disagreement between nodes.
+
+.. code:: sh
+
+       nodetool describecluster
+
+Drain the nodes and backup the data
+-------------------------------------
+
+Before any major procedure, like an upgrade, it is recommended to backup all the data to an external device. In ScyllaDB, backup is done using the ``nodetool snapshot`` command. For **each** node in the cluster, run the following command:
+
+.. code:: sh
+
+   nodetool drain
+   nodetool snapshot
+
+Take note of the directory name that nodetool gives you, and copy all the directories having this name under ``/var/lib/scylla`` to a backup device.
+
+When the upgrade is completed on all nodes, the snapshot should be removed with the ``nodetool clearsnapshot -t <snapshot>`` command, or you risk running out of space.
+
+
+Backup the configuration files
+--------------------------------
+
+.. code:: sh
+
+   for conf in $(cat /var/lib/dpkg/info/scylla-*server.conffiles /var/lib/dpkg/info/scylla-*conf.conffiles /var/lib/dpkg/info/scylla-*jmx.conffiles | grep -v init ) /etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; do sudo cp -v $conf $conf.backup-5.0; done
+
+Gracefully stop the node
+------------------------
+
+.. code:: sh
+
+   sudo service scylla-server stop
+
+Download and install the new release
+------------------------------------
+Before upgrading, check what version you are running now using ``dpkg -l scylla\*server``. You should use the same version in case you want to |ROLLBACK|_ the upgrade. If you are not running a 5.0.x version, stop right here! This guide only covers 5.0.x to 2022.1.y upgrades.
+
+To upgrade:
+
+#. Update the |APT_ENTERPRISE|_ to **2022.1**.
+
+#. Install:
+
+    .. code:: sh
+
+       sudo apt-get clean all
+       sudo apt-get update
+       sudo apt-get remove scylla\*
+       sudo apt-get install scylla-enterprise
+       for conf in $(cat /var/lib/dpkg/info/scylla-*server.conffiles /var/lib/dpkg/info/scylla-*conf.conffiles /var/lib/dpkg/info/scylla-*jmx.conffiles | grep -v init ) /etc/systemd/system/{var-lib-scylla,var-lib-systemd-coredump}.mount; do sudo cp -v $conf.backup-5.0 $conf; done
+       sudo systemctl daemon-reload
+
+    Answer ‘y’ to the first two questions.
+
+    If you installed version 5.0 using a ScyllaDB image (for example, AMI) or first installed an earlier version of ScyllaDB 
```

Comment:
I'd rather rephrase it to:
'If you use cloud image with preinstalled version of Scylla, install additional package  `scylla-enterprise-machine-image` with command `sudo apt-get install scylla-enterprise-machine-image`'

this is for all debian/ubuntu guides.

For guides related to centos, command is different: `sudo yumt install scylla-enterprise-machine-image` (yum instead of apt-get).

Reply:
@soyacz Thanks, I've rephrased the paragraph.
The information CentOS is in this file: docs/upgrade/upgrade-to-enterprise/upgrade-guide-from-5.0-to-2022.1/upgrade-guide-from-5.0-to-2022.1-rpm.rst
The information for Ubuntu and Debian is in the common file: docs/upgrade/_common/upgrade-guide-from-5.0-to-2022.1-ubuntu-and-debian.rst.
The file is included in all the pages for specific Ubuntu and Debian versions.